docs/source/spec-test-format.rst

```diff
@@ -130,17 +130,21 @@ A Test Scenario File has the following keys:
   be exactly one ``loop`` operation per scenario, and it SHOULD be the last
   operation in the scenario.
 
-  The scenario MUST use ``storeErrorsAsEntity``, ``storeFailuresAsEntity``,
-  ``storeSuccessesAsEntity`` and ``storeIterationsAsEntity`` operation arguments
-  to allow the workload executor to retrieve errors, failures and operation
-  counts for the executed workload. The entity names for these options MUST
-  be as follows:
-  
+  The scenario MUST use ``storeErrorsAsEntity``, ``storeSuccessesAsEntity``,
+  and ``storeIterationsAsEntity`` operation arguments to allow the workload
+  executor to retrieve errors, failures, and operation and iteration counts for
+  the executed workload. The entity names for these options MUST be as follows:
+
   - ``storeErrorsAsEntity``: ``errors``
-  - ``storeFailuresAsEntity``: ``failures``
   - ``storeSuccessesAsEntity``: ``successes``
   - ``storeIterationsAsEntity``: ``iterations``
-  
+
+  The scenario MUST NOT use ``storeFailuresAsEntity`` to ensure that all errors
+  and failures are reported under a single ``errors`` entity irrespective of how
+  a test runner might distinguish errors and failures (if at all). Note that
+  some ValidateWorkloadExecutor tests may still use ``storeFailuresAsEntity``
+  with the entity name ``failures`` to assert workload executor correctness.
```

Comment:
IIRC, when @p-mongo was implementing the unified test runner changes we concluded that the actual scenario tests should only use `storeErrorsAsEntity` so that Astrolabe could reliably consult `errors` and `numErrors` without having to worry about how drivers might distinguish errors and failures (assuming they do at all).

Since we still test `storeFailuresAsEntity` in the validation tests, I thought that deserved a mention here.